### DISCUSSION TITLE
Using skypilot in CI/CD to automate training

### DISCUSSION THREAD

**Discussion by merryHunter** [skypilot-org/skypilot#3931] [Q&A] [answered]
Hi,

I would like to use skypilot job on AWS spot instances to have DL models trained on GPU. Importantly, I want it to be inside a GitHub Action as past of CI/CD and run on schedule. Is there anything that may prevent me from having such setup? Or any suggestions, best practices to follow?

**Comment by concretevitamin** [comment, marked as answer]
Hey @merryHunter -- check out this minimum example I put together a while back: https://github.com/concretevitamin/skypilot-ci-cd-runner

**Reply by merryHunter** [reply]
Thanks, neat and great!

**Reply by concretevitamin** [reply]
Awesome, let us know if it worked?
